code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -502,6 +511,14 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "tequila Sunrise"
 	glass_desc = "Oh great, now you feel nostalgic about sunrises back on Terra..."
 
+/datum/reagent/consumable/ethanol/tequila_sunrise/on_mob_add(mob/living/M)
+	to_chat(M, "<span class='notice'>You feel gentle warmth spread through your body!</span>")
+	M.set_light(3, 0.7, "#FFCC00") //Tequila Sunrise makes you radiate dim light, like a sunrise!
```

Comment:
please make this add a fake light object to the person instead of setting their light, since the mob might want its own light and you should stack them anyway